_inc/lib/admin-pages/class.jetpack-my-jetpack-page.php

```diff
@@ -1,187 +0,0 @@
-<?php
```

Comment:
In the past we've sometimes avoided to delete PHP files, and instead left them as blank to avoid fatals on some sites upon update. See 8c3790d8cb1463a687a250b6b3d687c03a0365ca for example, or 46bacdd32a56b6f0430f0b128694e57df8402d9c. Should we do the same here?